lib/qme/quality_report.rb

```diff
@@ -167,7 +167,8 @@ def patient_results
     end
 
     def measure
-      QME::QualityMeasure.where({"hqmf_id"=>self.measure_id, "sub_id" => self.sub_id}).first
+      HealthDataStandards::CQM::Measure.where(hqmf_id: self.measure_id, sub_id: self.sub_id).first
```

Comment:
should we add a note like, 
For EcqmApiV2, we use HealthDataStandards::CQM::Measure as a main measure model instead of QME::QualityMeasure

Reply:
Yeah, I left this pretty open.  I wasn't sure how you wanted to show this.  I will add a comment.